openmc/tallies.py

```diff
@@ -304,24 +317,30 @@ def nonzero(val):
                 return 1 if not val else val
 
             # Reshape the results arrays
-            new_shape = (nonzero(self.num_filter_bins),
-                         nonzero(self.num_nuclides),
-                         nonzero(self.num_score_bins))
-
-            sum = np.reshape(sum, new_shape)
-            sum_sq = np.reshape(sum_sq, new_shape)
+            sum = np.reshape(sum, self.shape)
+            sum_sq = np.reshape(sum_sq, self.shape)
```

Comment:
I'm pretty sure I'm looking at the correct changeset (accounting for the fact this is based off of #525) and it still shows these as being changed in your PR only. I also explicitly looked at #525 and don't see these changes there.